Calculator.java

```diff
@@ -0,0 +1,47 @@
+import java.util.Scanner;
+
+public class Calculator {
+
+	static int calculate(int first, String symbol, int second) {
+		int result = 0;
+		if (symbol.equals("+")) {
+			result = first + second;
+
+		} else if (symbol.equals("-")) {
+			result = first - second;
+
```

Comment:
```suggestion
		if (symbol.equals("+")) 
			return first + second;

		if (symbol.equals("-")) 
			return first - second;

```

자바에서 for문이나 if문은 내용이 한줄이면 중괄호과 필요 없다는 점과 result를 저장해 마지막에 return 하는 것이 아닌 early return으로 더 간결하게 표현할 수 있을거 같아요!